### DISCUSSION TITLE
Timer

### DISCUSSION THREAD

**Discussion by Vic55105** [CubeLabsNZ/CubeTime#46] [Ideas]
I was wondering if y’all could add a manual input mode  for those that still want to use a stack Mat but still want to view stats. This would be greatly appreciated. 
(and one more thing good job on the app love it so much)

**Comment by pdtxie** [comment]
Swiping down on the timer will bring up the manual input option. Press the '+' icon to enter in a time.
As for a dedicated input mode (like csTimer), it is coming! We've been working on it, and should be introduced in the next couple of updates.
Thanks :)

**Comment by Vic55105** [comment]
Thank you so much

Sent from my iPhone

On May 1, 2022, at 20:09, Tim ***@***.***> wrote:

﻿

Swiping down on the timer will bring up the manual input option. Press the '+' icon to enter in a time.
As for a dedicated input mode (like csTimer), it is coming! We've been working on it, and should be introduced in the next couple of updates.
Thanks :)

**Comment by pdtxie** [comment]
dedicated input mode added in v2 releasing today
